apps/blockchain_web/lib/blockchain_web/endpoint.ex

```diff
@@ -47,8 +47,12 @@ defmodule Blockchain.Web.Endpoint do
   It receives the endpoint configuration from the config files
   and must return the updated configuration.
   """
-  def load_from_system_env(config) do
-    port = System.get_env("PORT") || raise "expected the PORT environment variable to be set"
-    {:ok, Keyword.put(config, :http, [:inet6, port: port])}
+  def init(_key, config) do
+    if config[:load_from_system_env] do
+      port = System.get_env("PORT") || 4000
```

Comment:
I am lost, why can you put this on the configs? This is silly IMO

Reply:
To be honest, I blindly followed warnings instruction I got from phoenix when building the app as a release. And I'm still confused about Phoenix configs, never took the time to really look into them correctly ...

I merged this PR to quickly sorry !! Happy to see your comments anyway, here is the next "big thing" that's on my plate https://github.com/robinmonjo/coincoin/issues/9. What do you think ?

Hope I will find time to work on this soon